locales/it/index.yml

```
@@ -505,8 +505,14 @@ authentication:
       incorrectCiePinContent1: Per la tua sicurezza puoi provare ad inserire il PIN della carta non più di tre volte. Poi dovrai sbloccare la carta inserendo il PUK.
       incorrectCiePinContent2: Ricorda che il codice PIN della tua Carta d'Identità Elettronica (CIE) è composto da 8 cifre. Quattro cifre che ti sono state consegnate all'anagrafe e altre quattro cifre che ti sono state spedite a casa, insieme alla carta.
       alert: Vuoi davvero tornare indietro?
+      subtitleHelp: Il PIN è un codice di 8 cifre.
+      subtitleCTA: Dove lo trovo?
+      bottomSheetTitle: Dove trovo il mio PIN?
+      bottomSheetText: "Il codice PIN è composto da 8 cifre:\n• le prime 4 cifre ti sono state consegnate all’anagrafe;\n• le ultime 4 cifre ti sono state spedite a casa, insieme alla carta d’identità elettronica.\n**Mettile insieme e inseriscile in quest’ordine.**"
```

Since there is a bug in the `Markdown` component, substitute the `\n` with the `\n\n`
```suggestion
      bottomSheetText: "Il codice PIN è composto da 8 cifre:\n\n• le prime 4 cifre ti sono state consegnate all’anagrafe;\n\n• le ultime 4 cifre ti sono state spedite a casa, insieme alla carta d’identità elettronica.\n\n**Mettile insieme e inseriscile in quest’ordine.**"
```